app/controllers/practices_controller.rb

```diff
@@ -28,9 +28,16 @@ def index
     redirect_to root_path
   end
 
+  def is_enabled
```

Comment:
in the rails world, usually we would use a `before_action` 
(examples at the top of the file)
so it would be
```
before_action :is_enabled, only: [:show, <other actions here if needed>]
```
then you can remove line 40 and we can add actions to the list if we need to

Reply:
got it.  made this change as well.